AgendaUWP/AgendaUWP/Views/AddContactPage.xaml

```diff
@@ -0,0 +1,51 @@
+﻿<Page
+    x:Class="AgendaUWP.Views.AddContactPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:AgendaUWP"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    xmlns:prism="using:Prism.Windows.Mvvm"
+    prism:ViewModelLocator.AutoWireViewModel="true"
+    >
+    <Grid Margin="10">
+        <Grid.Background>
+            <LinearGradientBrush StartPoint="0.5,0" EndPoint="0.5,1">
+                <GradientStop Color="White" Offset="0.0" />
+                <GradientStop Color="Blue" Offset="1.0" />
+            </LinearGradientBrush>
+        </Grid.Background>
+
+        <CommandBar Background="LightGray"  HorizontalAlignment="Stretch" Margin="0,10,0,10" OverflowButtonVisibility="Visible" >
+            <AppBarToggleButton Icon="Save" Label="Save"  VerticalAlignment="Center" />
+            <AppBarSeparator/>
+            <AppBarToggleButton Icon="Cancel" Label="Cancel"  VerticalAlignment="Center" Command="{Binding CancelSaveCommand}" />
+            <CommandBar.Content>
+                <TextBlock Text="Contacts"  VerticalAlignment="Center" HorizontalAlignment="Center" Margin="0,10" Foreground="Black"/>
+            </CommandBar.Content>
+        </CommandBar>
+
+        <Grid HorizontalAlignment="Center" VerticalAlignment="Center">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="auto" />
+                <RowDefinition Height="auto"/>
+                <RowDefinition Height="auto"/>
+                <RowDefinition Height="auto"/>
+                <RowDefinition Height="auto"/>
+            </Grid.RowDefinitions>
+
+            <TextBlock Text="Contact" Margin="0,15" FontSize="30"/>
+            <TextBox Grid.Row="1" Margin="0,5" PlaceholderText="Full Name" FontSize="20" TextWrapping="Wrap" Width="350" />
```

Comment:
Please, consider creating a style for this TextBox to avoid multiple properties identical